source/rooms/battle-gaiaknightessandelegantstella.js

```diff
@@ -2,11 +2,11 @@ const { RoomTemplate } = require("../classes");
 const { parseExpression } = require("../util/mathUtil");
 const { generateCombatRoomBuilder } = require("../util/messageComponentUtil");
 
-const enemies = [["Gaia Knightess", "1"], ["Elegant Stella", "1"], ["Asteroid", "0.25*n-1"]];
+const enemies = [["Gaia Knightess", "1"], ["Elegant Stella", "1"], ["Asteroid", "0.34*n-1"]];
```

Comment:
Does this 11% buff hit a certain break point?

Reply:
intuition says that I'd like it to be the same enemy scaling as I'm having the other knight encounters I was thinking
3 enemies to 3 delvers. But also, Gaia already makes ads so I felt like it could be toned one step back. Eh, the asteroid is more disposable to the point where it probably would be fine at .5 scaling (with initial setback to account for stella)